src/api/db/migrate/20170619111734_alter_notifications_to_use_polymorphic.rb

```diff
@@ -4,7 +4,13 @@ def change
 
     Notification.find_in_batches batch_size: 500 do |batch|
       batch.each do |notification|
-        notification.subscriber = notification.user || notification.group
+        if notification.user_id
+          notification.subscriber_type = 'User'
```

Comment:
@evanrolfe the problem comes from the old data, because now we don't have the **user** or **group** relationships, so we don't have a model instance to assign.